src/messages/MMonMap.h

```diff
@@ -48,11 +55,17 @@ class MMonMap final : public Message {
 
     using ceph::encode;
     encode(monmapbl, payload);
+    encode(peer_addrs, payload, features);
+    encode(quorum, payload);
+    encode(epoch, payload);
```

Comment:
I think including quorum information in this map subscription is the right approach.